test/python/circuit/test_gate_definitions.py

```diff
@@ -338,7 +340,7 @@ class TestStandardEquivalenceLibrary(QiskitTestCase):
         CU3Gate,
         XGate,
         CXGate,
-        ECRGate,
+        ECRGate,ough.  You might be interested (wr
```

Comment:
rogue middle-mouse click?

Reply:
Yeah I guess, I'm not really sure. I did the rebase through the web ui and I guess my finger slipped. It's fixed in: https://github.com/Qiskit/qiskit-terra/pull/10314/commits/a788da48653da22421542223c3312cc0ad55d193